Update Cmake to latest OV libs

preetha-intel opened the pull request.

sfatimar reviewed (commented):
@preetha-intel  can u please fix cpp lint errors so that it is easier to review the code.

sfatimar reviewed (commented):
Can you add correct copyright information  in the files you have changed. And add comments on Dynamic Shapes. What are the unit tests for this change

preetha-intel commented:
> @preetha-intel can u please fix cpp lint errors so that it is easier to review the code.



> @preetha-intel can u please fix cpp lint errors so that it is easier to review the code.



> Can you add correct copyright information in the files you have changed. And add comments on Dynamic Shapes. What are the unit tests for this change

Fixed Lint and added comments. Copyright needs to be updated from 2019-2022 to 2019-2024 on all the files for OpenVINO EP. I shall add as a seperate PR for cleaner approach.
